### ISSUE TITLE
Game - Level Editor

### ISSUE THREAD

**Issue by Nazariglez** [Nazariglez/primegame#66]

**Comment by mundusnine** [comment]
Since you want to support CastleDB tiles shouldnt you just use caslteDB's [map editor](http://castledb.org/#level) ?

**Comment by Nazariglez** [comment]
Yes, i'll support their map editor, it's awesome. This issue it's just a reminder to me for a "test" i want to do, a full level editor integrated with the code, displaying entities and componentes in it, (like gamemaker for example). Editors like castledb or tiled are great, but works parsing ids from the editor in your code, and create something more specific for the engine seems something fun to do.

**Comment by mundusnine** [comment]
Ok. If ever you want to use a UI editor in kha to make the level editor you can have a look at armorUI or haxe-UIkha( if you dont know them already that is :P) For armorUI if you find it too limited you could have a look at my fork which I hope could be eventually added to the master if ever it fits with the view that lubos has of armorUI. Looking forward to gecko :)
